README.md

```diff
@@ -19,12 +19,12 @@
    ```
    ````
 1. Write the post and select appropriate categories. Tags are not available.
-1. Add required packages to DESCRIPTION file via `usethis::use_package(<package>)`.
-1. Preview locally using `rmarkdown::render_site()` and `pkgdown::preview_site()` or (when using RStudio) by clicking on "Build Website" in the "Build" pane.
```

Comment:
Why should this be removed? This should still work and the way to do it to preview a draft post if a local Rmd has been rendered.

Reply:
yes, makes sense. Fixed this.